tests/aws/services/stepfunctions/v2/scenarios/test_base_scenarios.py

```diff
@@ -1675,9 +1675,7 @@ def test_map_item_reader_csv_max_items(
         sfn_snapshot.add_transformer(RegexTransformer(bucket_name, "bucket-name"))
 
         key = "file.csv"
-        csv_file = (
-            "Col1,Col2\n" "Value1,Value2\n" "Value3,Value4\n" "Value5,Value6\n" "Value7,Value8\n"
-        )
+        csv_file = "Col1,Col2\nValue1,Value2\nValue3,Value4\nValue5,Value6\nValue7,Value8\n"
```

Comment:
This is harder to understand now.

Reply:
Thanks for the input! You could disable this by disabling the specific formatting rule for this section (I think this should be ISC001). I would still move forward with this now to get this PR over the line :)